FTB/Signatures/CrashInfo.py

```diff
@@ -1041,10 +1041,10 @@ def isDerefOp(op):
             return "(" in op and ")" in op
 
         def calculateDerefOpAddress(derefOp):
-            match = re.match("\*?((?:\\-?0x[0-9a-f]+)?)\\(%([a-z0-9]+)\\)", derefOp)
+            match = re.match("\\*?((?:\\-?0x[0-9a-f]+)?)\\(%([a-z0-9]+)\\)", derefOp)
```

Comment:
Were these regexes broken before? We should have plenty of test coverage on these. Or does Python implicitly do the right thing here and this doesn't change semantics at all?